include/hyperpose/utility/data.hpp

```diff
@@ -18,9 +19,11 @@ struct feature_map_t {
     /// Constructor.
     /// \param name Tensor name. (Often from DNN engine graphs)
     /// \param tensor Tensor data.
-    inline feature_map_t(std::string name, ttl::tensor<float, 3> tensor)
+    /// \param shape Shape of tensor. (no batch dimension)
+    inline feature_map_t(std::string name, std::unique_ptr<char[]>&& tensor, std::vector<int> shape)
```

Comment:
use `std::array<int, 3>` for shape

Reply:
> use `std::array<int, 3>` for shape

I made this change cause I am implementing pose proposal network's post processing, where has different dim requirement.

One of the output tensor has a dim of [17, 9, 9, 12, 12].